packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -67,22 +65,24 @@ const Dashboard = ({ active_tab, setActiveTab, toggleStrategyModal, is_drawer_op
                     </Tabs>
                 </div>
             </div>
-            <div className='dashboard__run-strategy-wrapper'>
-                <RunStrategy />
-                {/*
-                    1. Bot-Builder
-                    2. Quick Strategy
-                    3. Charts
-                */}
-                {[1, 2, 3].includes(active_tab) && <RunPanel />}
-            </div>
-        </>
+            <DesktopWrapper>
+                <div className='dashboard__run-strategy-wrapper'>
+                    <RunStrategy />
+                    {/*
+                        1. Bot-Builder
+                        2. Quick Strategy
+                        3. Charts
+                    */}
+                    {[1, 2, 3].includes(active_tab) && <RunPanel />}
```

Comment:
the problem here is, if we do it this way we have to extra coercion, because the active_tab is a number and the Object.keys return array of strings (keys). so thats why can we keep it as it is as of now. @sara-fs

Reply:
so if the tabs are numbers you can do something like this
```

const tabs = Object.freeze({
   "first tab":1,
  "second tab":2,
   "third tab":3,
});
Object.values(tabs).includes(active_tab);
```

Reply:
We will make the tab index to string indexes such as `dashboard`, 'charts' etc and will be using those instead of default number indexes/